modules/connectivity/ParticipantConnectionStatus.js

```diff
@@ -18,10 +18,55 @@ const logger = getLogger(__filename);
 const DEFAULT_RTC_MUTE_TIMEOUT = 2000;
 
 /**
+ * The time to wait a track to be restored. Track which was out of lastN
+ * should be inactive and when entering lastN it becomes restoring and when
+ * data is received from bridge it will become active, but if no data is
+ * received for some time we set status of that participant connection to
+ * interrupted.
+ * @type {number}
+ */
+const DEFAULT_RESTORING_TIMEOUT = 5000;
+
+/**
+ * Participant connection statuses.
+ *
+ * @type {{
+ *      ACTIVE: string,
+ *      INACTIVE: string,
+ *      INTERRUPTED: string,
+ *      RESTORING: string
+ * }}
+ */
+export const ParticipantConnectionStatus = {
+    /**
+     * Status indicating that connection is currently active.
+     */
+    ACTIVE: Symbol('active'),
```

Comment:
I hope you don't hate me... turns out pollyfilling Symbol globally for Jitsi Meet is a bit cumbersome at the moment. Can you go back to strings? Sorry! 😢